### PR TITLE
Fix runtime exception in ImageClassification.

### LINKED ISSUE #3248: Fix runtime exception in Image classification sample.
This sample throws an exception for two reasons 1) model not found, should download the mode and unzip 2) even if the model exist it needs to set addBatchDimensionInput to true.

### PR DESCRIPTION
**Pull request by codemzs**
fixes #3248

### CODE REVIEW

File: docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/ImageClassification.cs [hunk 1]
```diff
@@ -86,5 +97,31 @@ class OutputScores
...
+            using (WebClient client = new WebClient())
+            {
+                client.DownloadFile(new Uri($"{baseGitPath}"), dataFile);
+            }
+
+            return dataFile;
+        }
+
+        /// <summary>
+        /// Taken from https://github.com/icsharpcode/SharpZipLib/wiki/GZip-and-Tar-Samples.
```
Comment (zeahmed) [inline docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/ImageClassification.cs]: >https://github.com/icsharpcode/SharpZipLib/wiki/GZip-and-Tar-Samp [](start = 23, length = 65)

Don't we have anything in .NET for zip/unzip. Are we fine to use external libraries?

The link is for wiki section. I think it should be for main repo, right?

Comment (codemzs) [reply docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/ImageClassification.cs]: Nope. I tried but this is the most talked library.

File: docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/ImageClassification.cs [hunk 2]
```diff
@@ -1,5 +1,9 @@
 using System;
+using System.IO;
 using System.Linq;
+using System.Net;
+using ICSharpCode.SharpZipLib.GZip;
```
Comment (wschin) [inline docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/ImageClassification.cs]: Just make sure its license is good. :)

Comment (codemzs) [reply docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/ImageClassification.cs]: Done.

**Pull request merged**